lib/Mongo/MongoClient.php

```diff
@@ -376,6 +376,8 @@ private function extractUrlOptions($server)
             $keyValue = explode('=', $option);
             if ($keyValue[0] === 'readPreferenceTags') {
                 $options[$keyValue[0]][] = $this->getReadPreferenceTags($keyValue[1]);
+            } elseif (ctype_digit($keyValue[1])) {
```

Comment:
This requires adding `ext-ctype` to composer.json, as was done with `ext-hash`.

Reply:
I've added the dependecy to `composer.json`